aiosmtpd/tests/test_smtp.py

```diff
@@ -434,6 +434,13 @@ def test_mail_params_bad_syntax_esmtp(self):
                 response,
                 b'Syntax: MAIL FROM: <address> [SP <mail-parameters>]')
 
+    def test_mail_fail_parse_email(self):
+        with SMTP(*self.address) as client:
+            client.helo('example.com')
+            code, response = client.docmd('MAIL FROM: <""@example.com>')
+            self.assertEqual(code, 501)
+            self.assertEqual(response, b'Syntax: MAIL FROM: <address>')
```

Comment:
Why? Sure, recipient of this response knows, what was send. That will only create possible encoding problems.